ipaclient/install/client.py

```diff
@@ -2126,6 +2126,13 @@ def install_check(options):
         logger.warning("Option 'force-join' has no additional effect "
                        "when used with together with option 'keytab'.")
 
+    # Remove empty keytab file
+    if (
```

Comment:
Could you please use `\` instead of `()` ?using  brackets do feel troublesome to me.
```python
if os.path.isfile(paths.KRB5_KEYTAB) and \
        os.path.getsize(paths.KRB5_KEYTAB) == 0 :
    remove_file(paths.KRB5_KEYTAB)
```

Reply:
Tibor, I think tests are failing for unrelated reasons, like Flo mentioned later, this test is not part of nightly bundle.

About breaking lines, I'm using [black](https://github.com/ambv/black) as reference, it suggests the use of parentheses in cases like this. However, after my changes this is not necessary anymore, so we are good! :-)